src/main/java/org/mskcc/cbio/portal/dao/DaoClinicalEvent.java

```diff
@@ -202,6 +203,26 @@ public static void deleteByCancerStudyId(int cancerStudyId) throws DaoException
             JdbcUtil.closeAll(DaoClinicalEvent.class, con, pstmt, rs);
         }
     }
+
+    public static void deleteByPatientId(int patientId) throws DaoException {
+        Connection con = null;
+        PreparedStatement pstmt = null;
+        ResultSet rs = null;
+        try {
+            con = JdbcUtil.getDbConnection(DaoClinicalEvent.class);
+
+            pstmt = con.prepareStatement("DELETE clinical_event, clinical_event_data" +
```

Comment:
Code can be simplified: 
- `clinical_event_data` is not needed here because there is already a FK cascade constraint: https://github.com/cBioPortal/cbioportal/blob/387509c3250a72c3c60b7a3d545f97aac218d7c2/src/main/resources/db-scripts/cgds.sql#L668

Reply:
see here https://github.com/cBioPortal/cbioportal-core/pull/47/commits/9089e77f277211c0ca1aeaa81ddb284cddeeadfd